packages/react-vapor/src/components/tab/tests/TabSelectors.spec.tsx

```diff
@@ -47,5 +47,16 @@ describe('TabSelector', () => {
             const expectedTabGroup = tabGroupState.tabs[0];
             expect(tabGroup).toEqual(expectedTabGroup);
         });
+        it('should return true if a tab is selected, false if not', () => {
```

Comment:
It should be outside the describe starting on [line 8.](https://github.com/coveo/react-vapor/pull/1622/files#diff-1687ca765e3cd4c51dfed88d055bb27eR8)

Reply:
woops